.gitignore

```diff
@@ -22,3 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+src/stories
```

Comment:
- [ ] Why is storybook stuff getting added to .gitignore

Reply:
not sure if the storybook files were meant to get pushed up the remote repo soz, not sure what it even is i just ran the installation as part of the instructions in the readme's